dashboard/modules/metrics/grafana_dashboard_factory.py

```diff
@@ -380,6 +412,14 @@ class Panel:
             "alias": "/FINISHED|FAILED|DEAD|REMOVED/",
             "hiddenSeries": True,
         },
+        {
+            "$$hashKey": "object:2987",
+            "alias": "MAX + PENDING",
+            "dashes": True,
+            "color": "#ff9a00",
```

Comment:
Hmm looking again at the chart, we are probably adding too many different colors. How about making it light gray for now to avoid looking too busy?

Reply:
<img width="592" alt="Screen Shot 2022-12-19 at 11 06 33 PM" src="https://user-images.githubusercontent.com/18510752/208604385-660d1930-083c-40ea-908d-3ec1faeba0ff.png">

Chose #77777